operations/helm/charts/grafana-agent/CHANGELOG.md

```diff
@@ -10,9 +10,14 @@ internal API changes are not present.
 Unreleased
 ----------
 
+### Bugfixes
```

Comment:
Bugfixes goes below enhancements, doesn't it? https://github.com/grafana/agent/blob/main/docs/developer/contributing.md#updating-the-changelog